### PR TITLE
more import styling updates

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

make more tweaks for consistency with reports

| before | after |
|--|--|
| image | image |

### Reviewer guidance

look good?

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/device_management/assets/src/views/SelectContentPage/index.vue [hunk 1]
```diff
@@ -41,10 +41,7 @@
           v-if="transferredChannel && onDeviceInfoIsReady"
           class="updates"
         >
-          <div
-            v-if="newVersionAvailable"
-            class="updates-available"
-          >
+          <div v-if="newVersionAvailable">
```
Comment (jonboiser) [inline kolibri/plugins/device_management/assets/src/views/SelectContentPage/index.vue]: some tests might use this class as a selector

Comment (indirectlylit) [reply kolibri/plugins/device_management/assets/src/views/SelectContentPage/index.vue]: updated

**Review by jonboiser — APPROVED** [review]
:LGTM:

**Comment by rtibbles** [comment]
Still some failing tests in the frontend tests.

**Comment by indirectlylit** [comment]
@jonboiser @rtibbles  updated tests in 3a68fba94d953fc5d9a212aa7ed04232468b96c1

Curious what you think. Some goals I had:

* Keep the tests focused on what's mentioned in the `it` phrases
* Minimize use of non-semantic selectors: DOM structure and CSS classes
* Maximize use of semantic selectors: component names and HTML tags

Two things that turned out to be helpful which I didn't previously realize:

* `.text()` uses [`textContent`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) and recursively returns _all_ text, making deeper selectors unnecessary
* You can pass components into `find`

My sense more generally is that if we're finding ourselves writing complex selectors in Vue tests, that may be an indicator that the component should be broken into child-components that can either be individually tested or at least referenced in `find`.

**Pull request merged**